packages/vsce/src/utils/profileManagement.ts

```diff
@@ -216,13 +218,15 @@ export class ProfileManagement {
       try {
         const { response } = await getCache(session, { cacheToken: isPlex, nodiscard: false });
         if (response.records.cicscicsplex) {
-          for (const plex of toArray(response.records.cicscicsplex)) {
+          const cicscicsplexs = filterCicsplexByConstraints(toArray(response.records.cicscicsplex));
+
+          cicscicsplexs.forEach((value: {}, key: string) => {
             infoLoaded.push({
-              plexname: plex.plexname,
+              plexname: key,
               regions: [],
               group: false,
             });
-          }
+          });
```

Comment:
To loop only the keys, you could do something like
```
        [...cicscicsplexs.keys()].forEach((plexName: string) => {
            infoLoaded.push({
              plexname: plexName,
              regions: [],
              group: false,
            });
          });
```

Achieves the same though, so not crucial.